src/config_libconfig.c

```diff
@@ -566,6 +566,13 @@ char *parse_config_libconfig(options_t *opt, const char *config_file, bool *shad
 			goto err;
 		}
 	}
+	// --inactive-blur
+	if (lcfg_lookup_bool(&cfg, "inactive-blur", &opt->inactive_blur)) {
+		if (opt->blur_method == BLUR_METHOD_NONE) {
+			opt->blur_method = BLUR_METHOD_BOX;
```

Comment:
Same thing should be done in `options.c` too. Probably better to do this in one place after all config is parsed. Also, changing blue_method is not the right thing to do, what if the user doesn't want background blur?

Reply:
Moved the blur fallback code to picom.c in c586170, which seems to be the best place for it and fixes the segfault when using only CLI args.

Instead of overriding `blur_method`, it now uses a separate blur context. If `blur_method` is set then it copies `backend_blur_context`, otherwise a new context is created with the same box default as before.